### PR TITLE
Fix quota modal text for users

### PR DESCRIPTION
**Pull request by JammingBen**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8572

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/33580/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33580/tracing/copy-and-move-resources-in-personal-space-alice-2023-3-13-12-00-45.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33580/tracing/quick-link-alice-2023-3-13-12-02-26.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33580/tracing/quick-link-anonymous-2023-3-13-12-02-34.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33580/tracing/user-login-can-be-managed-in-the-admin-settings-admin-2023-3-13-11-59-34.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33580/tracing/user-login-can-be-managed-in-the-admin-settings-admin-2023-3-13-11-59-40.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33580/tracing/user-login-can-be-managed-in-the-admin-settings-alice-2023-3-13-11-59-39.zip 


</p></details>
:boom: The e2e-ocis tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/33585/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33585/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2023-3-13-02-09-23.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33585/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2023-3-13-02-09-53.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33585/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-carol-2023-3-13-02-09-59.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33585/tracing/re-sharing-alice-2023-3-13-02-12-44.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33585/tracing/re-sharing-brian-2023-3-13-02-12-32.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33585/tracing/re-sharing-carol-2023-3-13-02-12-37.zip 


</p></details>
:boom: The e2e-oc10 tests pipeline failed. The build has been cancelled.

File: packages/web-pkg/src/components/Spaces/QuotaModal.vue [hunk 1]
```diff
@@ -46,12 +47,87 @@ export default defineComponent({
...
+          count: props.spaces.length.toString()
+        })
+      }
+      return $gettext('Change quota')
+    })
+    const getSuccessMessage = (count: number) => {
+      if (props.resourceType === 'space') {
+        return $ngettext(
+          'Space quota was changed successfully',
+          'Space quota of %{count} spaces was changed successfully',
```
Comment (kulmann) [inline packages/web-pkg/src/components/Spaces/QuotaModal.vue:91]:
```suggestion
          'Quota of %{count} spaces was changed successfully',
```

File: packages/web-pkg/src/components/Spaces/QuotaModal.vue [hunk 2]
```diff
@@ -46,12 +47,87 @@ export default defineComponent({
...
+          count,
+          { count: count.toString() }
+        )
+      }
+      if (props.resourceType === 'user') {
+        return $ngettext(
+          'User quota was changed successfully',
+          'User quota of %{count} users was changed successfully',
```
Comment (kulmann) [inline packages/web-pkg/src/components/Spaces/QuotaModal.vue:99]:
```suggestion
          'Quota of %{count} users was changed successfully',
```

File: packages/web-pkg/src/components/Spaces/QuotaModal.vue [hunk 3]
```diff
@@ -46,12 +47,87 @@ export default defineComponent({
...
+          { count: count.toString() }
+        )
+      }
+      return $gettext('Quota was changed successfully')
+    }
+    const getErrorMessage = (count: number) => {
+      if (props.resourceType === 'space') {
+        return $ngettext(
+          'Failed to change space quota',
+          'Failed to change space quota for %{count} spaces',
```
Comment (kulmann) [inline packages/web-pkg/src/components/Spaces/QuotaModal.vue:110]:
```suggestion
          'Failed to change quota for %{count} spaces',
```

File: packages/web-pkg/src/components/Spaces/QuotaModal.vue [hunk 4]
```diff
@@ -46,12 +47,87 @@ export default defineComponent({
...
+          count,
+          { count: count.toString() }
+        )
+      }
+      if (props.resourceType === 'user') {
+        return $ngettext(
+          'Failed to change user quota',
+          'Failed to change user quota for %{count} users',
```
Comment (kulmann) [inline packages/web-pkg/src/components/Spaces/QuotaModal.vue:118]:
```suggestion
          'Failed to change quota for %{count} users',
```

File: packages/web-pkg/src/components/Spaces/QuotaModal.vue [hunk 5]
```diff
@@ -46,12 +47,87 @@ export default defineComponent({
...
+            name: props.spaces[0].name
+          })
+        }
+        return $gettext('Change quota for %{count} spaces', {
+          count: props.spaces.length.toString()
+        })
+      }
+      if (props.resourceType === 'user') {
+        if (props.spaces.length === 1) {
+          return $gettext('Change quota for user %{name}', {
```
Comment (kulmann) [inline packages/web-pkg/src/components/Spaces/QuotaModal.vue:77]:
```suggestion
          return $gettext('Change quota for user "%{name}"', {
```

File: packages/web-pkg/src/components/Spaces/QuotaModal.vue [hunk 6]
```diff
@@ -46,12 +47,87 @@ export default defineComponent({
...
   },
   emits: ['spaceQuotaUpdated'],
-  setup() {
+  setup(props) {
+    const { $gettext, $ngettext } = useGettext()
+
+    const modalTitle = computed(() => {
+      if (props.resourceType === 'space') {
+        if (props.spaces.length === 1) {
+          return $gettext('Change quota for space %{name}', {
```
Comment (kulmann) [inline packages/web-pkg/src/components/Spaces/QuotaModal.vue:67]:
```suggestion
          return $gettext('Change quota for space "%{name}"', {
```

**Review by kulmann — CHANGES_REQUESTED** [review]
Some nitpicks about strings. Hope those make sense. Otherwise good 👍

**Review by kulmann — APPROVED** [review]
nice 💪

**Pull request merged**